xml/dm_registers.xml

```diff
@@ -11,11 +11,18 @@
 
         Harts are nonexistent if they will never be part of this system, no
         matter how long a user waits. Eg. in a simple single-hart system only
-        one hart exists, and all others are nonexistent.
+        one hart exists, and all others are nonexistent. Debuggers may assume
+        that a system has no harts with ids higher than the first nonexistent
```

Comment:
This is a little misleading since I think you are mixing "id" with "sel value". "ID" was used pretty consistently to mean "the value you get when you read $mhartid" which could even go in descending order or something.

You could say "debuggers may assume than any harts selected by a hartsel value higher than the first nonexistent hartsel will all be nonexistent."